packages/http-client-csharp/generator/Microsoft.Generator.CSharp/perf/Microsoft.Generator.CSharp.Tests.Perf.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Microsoft.Generator.CSharp.ClientModel\src\Microsoft.Generator.CSharp.ClientModel.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Generator.CSharp.ClientModel\src\Microsoft.Generator.CSharp.ClientModel.Plugin.csproj" />
```

Comment:
this change doesn't look right it should be including the library its perf testing